External/FEXCore/Source/Interface/IR/Passes/RegisterAllocationPass.cpp

```diff
@@ -98,6 +98,14 @@ namespace {
     fextl::vector<SpillStackUnit> SpillStack;
     fextl::unordered_map<IR::NodeID, fextl::unordered_set<IR::NodeID>> BlockPredecessors;
     fextl::unordered_map<IR::NodeID, fextl::unordered_set<IR::NodeID>> VisitedNodePredecessors;
+
+    void *operator new(size_t size) {
```

Comment:
Leftover from previous version?

Reply:
Required due to raw new usage. Added a comment.